api/lib/domain/services/token-service.js

```diff
@@ -172,9 +173,10 @@ function extractClientId(token, secret = settings.authentication.secret) {
   return decoded.client_id || null;
 }
 
-function extractUserIdForCampaignResults(token) {
+function extractCampaignResultsTokenContent(token) {
   const decoded = getDecodedToken(token);
-  return decoded.access_id || null;
+  if (typeof decoded === 'boolean') return null;
```

Comment:
Pourquoi ne pas mettre une condition plus explicite qui correspond à la valeur renvoyée par getDecodedToken comme suit ?

```suggestion
  if (decoded === false) return null;
```

Reply:
Oui, tu as raison. Ce n'est pas nécessaire. Je fais le changement.